include/linux/soundwire/sdw_intel.h

```diff
@@ -119,6 +119,8 @@ struct sdw_intel_ops {
 			     struct sdw_intel_stream_params_data *params_data);
 	int (*free_stream)(struct device *dev,
 			   struct sdw_intel_stream_free_data *free_data);
+	int (*trigger)(struct snd_pcm_substream *substream,
+		       int cmd, struct snd_soc_dai *dai);
```

Comment:
other callbacks have device + some sort of struct as parameter, should this be aligned as well?